### ISSUE TITLE
Can we use NVDA for microsoft word ADD-INs?

### ISSUE THREAD

**Issue by Athinarapu-Akhila** [nvaccess/nvda#16733]
The issue is when i open the NVDA and its no focusable on the add-in itself( I.e., the blue focusable box is not focusing on the addin, which will appear at the right side pane

**Comment by Adriani90** [comment]
This is working as expected, when using tab key and arrow keys in the list of MS word add-ins. You can also focus them with the mouse.
Please describe your issue completely by filling the complete template, with exact steps to reproduce. Otherwise you can discuss your problem in the NVDA users list on groups.io.

Closing.
